style_query.css

```diff
@@ -1,10 +1,10 @@
 /* Font Imports */
-@import url('https://fonts.googleapis.com/css2?family=Russo+One&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
 
 /* Style for overall website */
 html{
-  background: #fff;
+  background: #121416;

```

Comment:
Majority consists of applying changing the background colour and font family from here and below.